compiler/ast/dag/op.go

```diff
@@ -101,10 +101,11 @@ type (
 		Paths []Seq  `json:"paths"`
 	}
 	Scope struct {
-		Kind   string  `json:"kind" unpack:""`
-		Consts []Def   `json:"consts"`
-		Funcs  []*Func `json:"funcs"`
-		Body   Seq     `json:"seq"`
+		Kind   string    `json:"kind" unpack:""`
+		Consts []Def     `json:"consts"`
+		Funcs  []*Func   `json:"funcs"`
+		UserOp []*UserOp `json:"user_ops"`
```

Comment:
Nit: The other slice field names are plural.
```suggestion
		UserOps []*UserOp `json:"user_ops"`
```